MIGRATION.md

```diff
@@ -0,0 +1,91 @@
+# Migrating to the `cloudant-python-sdk` library 
+This document is to assist in migrating from the `python-cloudant` (module: `cloudant`) to the newly supported [`cloudant-python-sdk`](https://github.com/IBM/cloudant-python-sdk) (module: `ibmcloudant`).
+
+## Initializing the client connection
+There are several ways to create a client connection in `cloudant-python-sdk`:
+1. [Enviornment variables](https://github.com/IBM/cloudant-python-sdk#authentication-with-environment-variables)
+2. [External configuration file](https://github.com/IBM/cloudant-python-sdk#authentication-with-external-configuration)
+3. [Programmatically](https://github.com/IBM/cloudant-python-sdk#programmatic-authentication)
+
+[See the README](https://github.com/IBM/cloudant-python-sdk#code-examples) for code examples on using enviornment variables.
+
+## Other differences
+1. The `cloudant-python-sdk` library does not support local dictionary caching of database and document objects.
+1. There's no context manager feature in `cloudant-python-sdk`.  You'll need to create the client, connect to the database, fetch and save a document as separate commands.
+1. In `cloudant-python-sdk` View, Search, and Query (aka `_find` endpoint) operation responses contain raw JSON content like using `raw_result=True` in `python-cloudant`.
+
+## Request mapping
+Here's a list of the top 5 most frequently used `python-cloudant` operations and the `cloudant-python-sdk` equivalent API operation documentation link:
+
+| `python-cloudant` operation           | `cloudant-python-sdk` API operation documentation link |
+|---------------------------------------|---------------------------------|
+|`Document('db_name', 'docid').fetch()` |[`getDocument`](https://cloud.ibm.com/apidocs/cloudant?code=python#getdocument)|
+|`db.get_view_result()`                 |[`postView`](https://cloud.ibm.com/apidocs/cloudant?code=python#postview)|
+|`db.get_query_result()`                |[`postFind`](https://cloud.ibm.com/apidocs/cloudant?code=python#postfind)|
+| `doc.exists()`                        |[`headDocument`](https://cloud.ibm.com/apidocs/cloudant?code=python#headdocument)|
+|`Document('db_name', 'docid').save()`  |[`putDocument`](https://cloud.ibm.com/apidocs/cloudant?code=python#putdocument)|
+
+
+[A table](#reference-table) with the whole list of operations is provided at the end of this guide.
+
+The beta SDKs are generated from a more complete API spec and provide a significant number of operations that did not exist in the legacy libraries. See [the IBM Cloud API Documentation](https://cloud.ibm.com/apidocs/cloudant) to review request parameter and body options, code examples, and additional details for every endpoint.
```

Comment:
Nit: what do you think of
```suggestion
The beta SDKs are generated from a more complete API spec and provide a significant number of operations that do not exist in `python-cloudant`. See [the IBM Cloud API Documentation](https://cloud.ibm.com/apidocs/cloudant) to review request parameter and body options, code examples, and additional details for every endpoint.
```
?

Reply:
yeah I meant to mention this, we should say "beta SDKs" either, because it will date when we move out of beta and talks about other SDKs that aren't strictly relevant here. 

The `cloudant-python-sdk` is generated...

Reply:
Ok, will make the changes.